src/routes/topUps.ts

```diff
@@ -8,12 +8,25 @@ import { getStripeClient } from "../helpers/stripe";
 import { getLogger } from "../logger";
 import * as backofficeHelpers from "./backoffice";
 import * as db from "../db";
-import { BookingType } from "../helpers/types";
+import { BookingType, TopUpStatus } from "../helpers/types";
 
 const log = getLogger("topUps");
 
 const TOP_UP_CHECK_DELAY_IN_MS = 3000;
 
+const mapTopUpStatus = (status: Stripe.PaymentIntent.Status): TopUpStatus => {
+  switch (status) {
+    case "canceled":
+      return TopUpStatus.CANCELLED;
+    case "processing":
```

Comment:
last time I've fetched not confirmed topup, I got status `requires_payment_method`. Maybe this one means ACCEPTED

Reply:
https://stripe.com/docs/payments/paymentintents/lifecycle#intent-statuses

Reply:
hmm but we create topup only after clicking the button, not on the card screen

Reply:
should we then map all of these `require_` to ACCEPTED?

Reply:
I think so